packages/kbn-apm-synthtrace/src/lib/logs/logs_synthtrace_es_client.ts

```diff
@@ -48,6 +48,33 @@ export class LogsSynthtraceEsClient extends SynthtraceEsClient<LogDocument> {
     }
   }
 
+  async createComponentTemplate(name: string, mappings: MappingTypeMapping) {
+    const isTemplateExisting = await this.client.cluster.existsComponentTemplate({ name });
+
+    if (isTemplateExisting) return this.logger.info(`Component template already exists: ${name}`);
+
+    try {
+      await this.client.cluster.putComponentTemplate({
+        name,
+        template: {
+          mappings,
+        },
+      });
+      this.logger.info(`Component template successfully created: ${name}`);
+    } catch (err) {
+      this.logger.error(`Component template creation failed: ${name} - ${err.message}`);
+    }
```

Comment:
Does it make sense to continue with test run when component template does not exist/failed to create?

Reply:
Well The only use case, where they will fail to create, would be either a Network issue or User roles and right issue. In that case, the test would fail.

Reply:
If there it is required to exist, I suggest throwing an error instead of just logging it. Thoughts?